org.adempiere.base/src/org/adempiere/util/ModelClassGenerator.java

```diff
@@ -168,6 +169,8 @@ private String createHeader (int AD_Table_ID, StringBuilder sb, StringBuilder ma
 		start.append("/** Generated Model for ").append(tableName).append(NL)
 			 .append(" *  @author iDempiere (generated) ").append(NL)
 			 .append(" *  @version ").append(Adempiere.MAIN_VERSION).append(" - $Id$ */").append(NL)
+			 .append("@IndexSubclasses").append(NL)
+			 .append("@org.adempiere.base.Model(table=\"").append(tableName).append("\", intermediate=true)").append(NL)

```

Comment:
with the new getSubclasses approach, isn't the intermediate attribute not needed anymore ?

Reply:
Sorted